### DISCUSSION TITLE
Questions regarding Velero

### DISCUSSION THREAD

**Discussion by coolprat999** [vmware-tanzu/velero#5757] [General discussions]
Hi Team,

Can someone provide details on below questions before starting with using Velero. 

i) Do we have the paid version of Velero or the Community edition with support? 
ii) How security updates are managed and released for Velero?

Thanks,

**Comment by coolprat999** [comment]
Hi Team,

Can someone please reply.

Thanks,

**Comment by coolprat999** [comment]
Hi All,

Can someone please respond? I am unable to find this information. 

Thanks,

**Comment by coolprat999** [comment]
Hi All,

Can someone please reply? Not sure why no one is responding. 

Due to this we are unable to respond back to client.

Thanks,

**Comment by draghuram** [comment]
Hi, I am not a core member of Velero team but we use Velero in our product so I will try to answer as best as I can. 

1) Velero itself is FOSS project so there is no concept of paid version or community edition with support. Having said that, you can use some commercial products that embed Velero for some or all of the functionality. CloudCasa is one example but there may be others.

2) As for security update, it would be best for a core team member to respond but from what I have seen, people usually open bugs when security problems are discovered (either in Velero or its dependencies). Velero team then releases new images with fixes. Again the answer in (1) may partially cover this area as well.

Hope this helps.

**Reply by coolprat999** [reply]
Hi @draghuram, 

Thank you for the reply.
